packages/dtx-web/src/app.css

```
@@ -3,8 +3,8 @@
 @import '@skeletonlabs/skeleton';
 @import '@skeletonlabs/skeleton/optional/presets';
 @import '@skeletonlabs/skeleton/themes/cerberus';
-@source "../../../node_modules/@skeletonlabs/skeleton-svelte/dist";
-@source "../../../node_modules/@dtx/common/dist";
+@source '../../../node_modules/@skeletonlabs/skeleton-svelte/dist';
```

Stylelint (17.3.0)

[error] Unexpected unknown at-rule "@source" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)